README.md

```diff
@@ -3,9 +3,10 @@ SublimeLinter-roslint
 
 This linter plugin for [SublimeLinter][docs] provides an interface to [roslint](http://wiki.ros.org/roslint). It will be used with files that have the “C++” syntax.
 
-This is for ROS Indigo currently.  You should be able to modify the indigo string inside things to make it work with Jade or other ROS releases. 
+This is tested in ROS Kinetic.
```

Comment:
and ROS indigo?

Reply:
I only tested in kinetic. I don't have an Indigo environment, sorry.